core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -208,6 +208,12 @@ export class ConversationSkinComponent {
       ReadOnlyExplorationBackendApiService
   ) {}
 
+  adjustPageHeightOnresize(): void {
+    this.windowRef.nativeWindow.onresize = () => {
+      this.adjustPageHeight(false, null);
+    };
+  }
+
```

Comment:
the function in writen outside because the frontend coverage checks are sometimes unable to cover this piece of code (detection issue) so I have written this piece of code outside `userService.getUserInfoAsync()` so that it can be tested without a flake